draft-ietf-ohai-ohttp.md

```diff
@@ -1120,12 +1122,35 @@ for ensuring the key configurations are consistent between different clients.
 
 # Operational and Deployment Considerations {#deployment}
 
+This section discusses various operational and deployment considerations.
+
+## Performance Overhead
+
 Using Oblivious HTTP adds both cryptographic and latency to requests relative to
 a simple HTTP request-response exchange.  Deploying proxy services that are on
 path between clients and servers avoids adding significant additional delay due
 to network topology.  A study of a similar system {{ODoH}} found that deploying
 proxies close to servers was most effective in minimizing additional latency.
 
+
+## Oblivious Proxy State {#proxy-state}
+
+The Oblivious Proxy Resource is a fixed mapping between URI and the Oblivious
+Request Resource. This means that any encapsulated request sent to the Oblivious
+Proxy Resource will always be forwarded to the Oblivious Request Resource.
+This constraint was imposed to simplify proxy configuration and mitigate against
+the Oblivious Proxy Resource being used as a generic proxy for unknown Oblivious
+Request Resources. The proxy will only forward for Oblivious Request Resources
+that it has explicitly configured and allowed.
+
+It is possible for an Oblivious Proxy to be configured with multiple Oblivious
```

Comment:
"Oblivious Proxy" isn't really a concept we use.

```suggestion
It is possible for a server to be configured with multiple Oblivious
```

Reply:
We use it on and off. I'll remove other references to it in the doc.